### PR TITLE
WIP FM-8269 add or remove features in inventory file

### PR DESCRIPTION
**Pull request by sheenaajay**

### CODE REVIEW

File: lib/puppet_litmus/inventory_manipulation.rb [hunk 1]
```diff
@@ -109,4 +109,65 @@ def remove_node(inventory_hash, node_name)
...
   end
+
+  # Adds a feature to the group specified/
+  #
+  # @param inventory_full_path [String] path to the inventory.yaml file
+  # @param feature_name [String] feature to locate in the group
+  # group_name [String] group of nodes to limit the search for the group_name in
+  # @return inventory.yaml file with feature added to group.
+  # @return [Hash] inventory_hash with feature added to group if group_name exists in inventory hash.
+  def add_to_inventory_file(inventory_full_path, feature_name, group_name)
```
Comment (tphoney) [inline lib/puppet_litmus/inventory_manipulation.rb]: should this become `add_feature_to_group` and `remove_feature_from_group` respectively

**Comment by tphoney** [comment]
The other inventory functions work on a hash, and dont read a file. should these new methods follow that ?

File: lib/puppet_litmus/inventory_manipulation.rb (hunk 1 above)
Comment (sheenaajay) [reply lib/puppet_litmus/inventory_manipulation.rb]: @tphoney Thanks for the comments.Will rename the files.

**Comment by sheenaajay** [comment]
@TP sure. That will be do able now. Since we moved the write part to a different function.Let me incorporate the comments. Thank you

**Comment by sheenaajay** [comment]
@tphoney Thanks for the valuable comments. All comments are incorporated.

**Comment by tphoney** [comment]
🆒 😲 🎆

**Pull request merged**